DTS-STN/Alpha-Site
# Analytics report problem

# Description

[Go through the Adobe Analytics Implementation Testing Process manual alongside the Implementation Run Testing Checklists](https://trello.com/b/ZqWtJSyh/alpha-site-board)

I added values to the report a problem checkboxes because they only returned true and that didn't help with analytics. I also updated the unsubscribe form for analytics, I hid the email field and added tracking for the submit button. I also a script component in projects pages for analytics.

## Test Instructions

1. Go in the report a problem component and check to see if each checkbox has a value
2. Make sure the submit button for report a problem has the analytics tracking prop
3. Check the unsubscribe page and make sure the email text field has "exclude" in the classname
4. Check the projects folder and make sure the pages have the "satellite.pageBottom()" script at the end of the page.

## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
